Cargo.toml

```diff
@@ -28,7 +28,9 @@ lto = true
 codegen-units = 1
 
 [features]
+default = []
 benchmarking = []
+js = ["wasm-bindgen"]
```

Comment:
The feature name `js` could cause confusion with JavaScript injection detection. Perhaps it could include wasm in the name?